### PR TITLE
Add LibreOffice Base

### LINKED ISSUE #9496: LibreOffice apps
### Brand Name

LibreOffice

- [x] LibreOffice Base #9595
- [ ] LibreOffice Calc #9923
- [x] LibreOffice Impress #9925 
- [x] LibreOffice Math #9926 
- [x] LibreOffice Writer #9927 

### Website

https://www.libreoffice.org/

### Popularity Metric

[GlobalRank](https://www.similarweb.com/website/libreoffice.org/#overview) equals 17,519

### Official Resources for Icon and Color

https://wiki.documentfoundation.org/Marketing/Branding

### Additional Comments

Since Microsoft Office has separate icons for Word, Excel, PowerPoint, Access, I think LIbreOffice could have separate icons for Writer, Calc, Impress, Base, Math

### Contributing

No

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
libreofficebase

**Issue:** closes #9496 

**Similarweb rank:** 16,861. See https://www.similarweb.com/website/libreoffice.org

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from local install
Color: `#7324a9` Taken from branding page https://wiki.documentfoundation.org/Design/Branding

### CODE REVIEW

**Comment by PratyushJayachandran** [comment]
Hello @simple-icons/icon-review!
I am creating this PR to start the discussion about the nitty gritties involved in adding all the LibreOffice icons.
Points I am confused about:

1. Which color to pick from https://wiki.documentfoundation.org/Design/Branding. Base or Shade?
2. What is the license type
3. How do we display source as I picked this up from the installed location (I tried but I couldn't find it in their git repo)
image

I have attached the source in #9496

**Comment by service-paradis** [comment]
You can find those SVGs on their Github mirror:
https://github.com/LibreOffice/help/tree/master/source/media/navigation

**Comment by PratyushJayachandran** [comment]
> You can find those SVGs on their Github mirror: https://github.com/LibreOffice/help/tree/master/source/media/navigation It looks like we can find them at https://opengrok.libreoffice.org/xref/core/sysui/desktop/icons/hicolor/scalable/apps/ too
> 
> For the color, I think I would opt for the base color.

Ahh finally! Thanks for getting the git.

**Comment by adamrusted** [comment]
Just looking at this, the gradient seems to use the 'Base' and 'Accent' colours. Given 'Shade' is generally a happy middle-ground, I'd be tempted to go with that for our hex. Either that, or take the two other colours mentioned and feed them into [MeyerWeb's Color Blender](https://meyerweb.com/eric/tools/color-blend/#::1:hex) to find the 'technical' midpoint.

**Comment by PratyushJayachandran** [comment]
> Just looking at this, the gradient seems to use the 'Base' and 'Accent' colours. Given 'Shade' is generally a happy middle-ground, I'd be tempted to go with that for our hex. Either that, or take the two other colours mentioned and feed them into [MeyerWeb's Color Blender](https://meyerweb.com/eric/tools/color-blend/#::1:hex) to find the 'technical' midpoint.

I chose the color in Base column. Upon comparison to the color blender you attached, the results are very close to each other. For simplicity, I would prefer to pick colors from their branding page.

If this PR seems fine, I will create other application logos in similar fashion.

**Comment by PratyushJayachandran** [comment]
libreofficebase

File: _data/simple-icons.json [hunk 1]
```diff
@@ -7571,6 +7571,15 @@
             "source": "https://wiki.documentfoundation.org/Marketing/Branding",
             "guidelines": "https://wiki.documentfoundation.org/Marketing/Branding"
         },
+        {
+            "title": "LibreOffice Base",
+            "hex": "B63AD3",
+            "source": "https://wiki.documentfoundation.org/Marketing/Branding",
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:7577]:
```suggestion
            "source": "https://github.com/LibreOffice/help/blob/e3b1cce7dde7e964c7670dd24a167e750654685a/source/media/navigation/libo-base.svg",
```

**Review by PeterShaggyNoble — APPROVED** [review]
Icon looks perfect to me 👍🏻 However, I would definitely interpret "Base" to mean the primary brand colour, so I think that's what we should use.

The source URL also need to be updated before merging.

**Pull request merged**